apps/condo/domains/notification/gql.js

```diff
@@ -24,7 +24,7 @@ const RESEND_MESSAGE = gql`
     }
 `
 
-const REMOTE_CLIENT_FIELDS = `{deviceId appId pushToken pushTransport devicePlatform pushType meta owner { id, type } ${COMMON_FIELDS}}`
+const REMOTE_CLIENT_FIELDS = `{deviceId appId pushToken pushTransport devicePlatform pushType meta pushTokenVoIP pushTransportVoIP pushTypeVoIP owner { id, type } ${COMMON_FIELDS}}`
```

Comment:
Does this mean that each client has different tokens for push and VoIP calls?

Reply:
For android pushTokens could be same, mobile apps will take care on that, for iOS they will be different.